agent/go/Makefile

```
@@ -45,11 +47,35 @@ vet: ## Run go vet against code.
+
+# Command and step tests re-execute covered test binaries. GOCOVERDIR prevents
+# runtime warnings from those child processes.
 .PHONY: unit-tests
-unit-tests: ginkgo ## Run Ginkgo/Gomega unit tests.
-	$(GINKGO) run ./...
+unit-tests: reporting ginkgo ## Run Ginkgo/Gomega unit tests with coverage.
+	mkdir -p $(COVERAGE_DATA_DIR)
+	GOCOVERDIR=$(abspath $(COVERAGE_DATA_DIR)) $(GINKGO) run --cover --coverprofile=unit.coverprofile --output-dir=$(REPORTING) ./...
+
+.PHONY: merge-coverage
+merge-coverage: reporting ## Merge test coverage profiles.
```

checkmake (0.3.2)

[warning] Target body for "merge-coverage" exceeds allowed length of 5 lines (9).

(maxbodylength)